mkt/extensions/urls.py

```diff
@@ -29,4 +31,7 @@
     url(r'search/$', ExtensionSearchView.as_view(), name='extension-search'),
     url(r'search/reviewers/$', ReviewerExtensionSearchView.as_view(),
         name='extension-search-reviewers'),
+    url(r'queue-update/$',
```

Comment:
nit, but I think queues-updates (and `TestReviewerExtensionUpdatesViewSet`, `ReviewerExtensionUpdatesViewSet` ...) would be slightly better. Initially I wondered why would add added an endpoint for updating things in the queue :)